boards/aitewinrobot-esp32c3-supermini/board.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "Aitewin Robot ESP32-C3 SuperMini",
+  "version": 1.8,
+  "description": "A common ESP32 dev board by AITEWIN ROBOT (Derived from Adafruit QT Py ESP32-C3)",
+  "author": "GeekGarage",
+  "mcu": "esp32-c3",
+  "fqbn": "esp32:esp32:esp32c3",
```

Comment:
Looks like this one is a better fit: https://github.com/espressif/arduino-esp32/commit/50f436c6fc3d525d08a81f6e0f222b88a7e6cc7d

It wasn't released yet, so let's keep the current value for now

Reply:
I'm quessing we are talking about the MCU and FQBN ?

Reply:
since we are keeping current values I'm guessing it is fixed for now. Let me know when the board is released and I will update